draft-ietf-quic-http.md

```diff
@@ -727,12 +727,14 @@ indicating different stream IDs, but MUST NOT increase the value they send in
 the last Stream ID, since clients might already have retried unprocessed
 requests on another connection.  A server that is attempting to gracefully shut
 down a connection SHOULD send an initial GOAWAY frame with the last Stream ID
-set to the maximum value allowed by QUIC's MAX_STREAMS and SHOULD NOT increase
-the MAX_STREAMS limit thereafter.  This signals to the client that a shutdown is
+set to the maximum value allowed by the concurrency control of QUIC for the
+client-initiated, bidirectional streams (see section 4.5 of {{QUIC-TRANSPORT}})
+or any value above that, and SHOULD NOT grant any more concurrency credit at the
```

Comment:
I would say "could request that the transport not grant any more ..." as this is does not need to carry normative force.  But that's not text you changed, so I'll leave that for @MikeBishop to sort out.

Reply:
:+1: FWIW, I agree to what @martinthomson states.

Reply:
PS. I think there is a bigger issue here, see https://github.com/quicwg/base-drafts/issues/3341#issuecomment-574956848.